provisioning/src/handlers/portfolioDrafts/get.ts

```diff
@@ -0,0 +1,18 @@
+import { Context } from '@azure/functions'
+import { CosmosClient } from '@azure/cosmos'
+
+export async function handler (context: Context): Promise<void> {
+  const connectionString = process.env['COSMOS-CONNECTION-STRING']
+  context.log('Connecting to ' + connectionString)
+  const client = new CosmosClient(connectionString)
+
+  // TODO: create these in terraform & fail fast
+  const { database } = await client.databases.createIfNotExists({ id: 'atat' })
+  const { container } = await database.containers.createIfNotExists({ id: 'portfolios' })
+
+  // TODO: add a modicum of error handling
+  const results = await container.items.readAll().fetchAll()
```

Comment:
We need to a slight transformation here to that the response matches the spec:

```
[
  {
      "id": "283c12eb-377f-4df5-82d0-5e553bab445c",
      "created_at": "{2021} 07-22T08:57:09 068 Z AM",
      "updated_at": "{2021} 07-22T08:57:09 068 Z AM",
      "status": "not_started"
  }
]
```

Right now it's wrapped by a `resources` property and also includes the cosmos metadata that we don't want to expose.

Reply:
That's exactly the type feedback I was seeking.  Thanks.

Reply:
A for loop to map each result to the ProvisioningStatus.ts model would be a good way to implement this.

Reply:
To expand on what zach is saying 

```
const results = await container.items.readAll().fetchAll().map((data) => {
      data.id,
      data.created_at
      data.updated_at,
      data.status
     })
```

Reply:
I miss the Java Streaming API....  :'(

Reply:
Very helpful. Thank you.